packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -158,15 +158,18 @@ type BoxProps = Partial<
     BoxVisualProps & { children?: React.ReactNode | React.ReactNode[] } & TestID
 >;
 
+type BaseBoxUtilityProps = Pick<CSSObject, 'touchAction' | 'userSelect' | 'opacity'>;
```

Comment:
we can add it to `BaseBoxVisualProps` only right?

Reply:
Sure should work.